AirBench/Controllers/ReviewController.cs

```diff
@@ -0,0 +1,58 @@
+﻿using AirBench.Data;
+using AirBench.FormModels;
+using AirBench.Models;
+using AirBench.Repositories;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.Web.Mvc;
+
+namespace AirBench.Controllers
+{
+    public class ReviewController : Controller
```

Comment:
I'd just add an `Authorize` attribute to the controller class instead of decorating each action method individually.